spec/factories/supplier.rb

```diff
@@ -3,5 +3,15 @@
 FactoryBot.define do
   factory :supplier do
     sequence(:name) { |n| "Test Supplier #{n}" }
+
+    trait :geoamey do
```

Comment:
good to have those I think but we aren't using them in this pr anymore, wonder if we should remove or keep them for future use

Reply:
I promise to use them in the future :-)